test/specs/components/Avatar/Avatar-test.tsx

```diff
@@ -4,4 +4,16 @@ import Avatar from 'src/components/Avatar/Avatar'
 
 describe('Avatar', () => {
   isConformant(Avatar)
+
+  describe('generateInitials', () => {
+    it('generateInitials should show just the initials of the first and last words in the name', () => {
+      expect(Avatar.defaultProps.generateInitials('John Middle Doe')).toEqual('JD')
+    })
+
+    it('generateInitials removes the text inside brackets', () => {
+      expect(Avatar.defaultProps.generateInitials('John Doe (Working position)')).toEqual('JD')
+      expect(Avatar.defaultProps.generateInitials('John Doe {Working position}')).toEqual('JD')
+      expect(Avatar.defaultProps.generateInitials('John Doe [Working position]')).toEqual('JD')
```

Comment:
This is a neat feature, we show it off with a dedicated example.  Maybe something like:

>### Excluded Initials
>Avatar initials exclude content in parens, braces, and brackets.